.github/workflows/github-workflows.yml

```diff
@@ -1,4 +1,4 @@
-name: Docker build and push to ghcr.io/ai-cfia and lint test
+name: Workflows
```

Comment:
python-backend-workflows? it would make it more readable in the Github Actions log interface perhaps?

Reply:
Alright, il'l do the same with all frontend workflow. Maybe something like `node-frontend-workflows`

Reply:
react-frontend-workflows would be more accurate IMO